modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -90,6 +90,19 @@ The resources allocated to your Mule application determine the application's per
 [NOTE]
 Run performance and load testing on your Mule applications to determine the number of resources to allocate.
 
+== Internal Load Balancer Memory Allocation
+
+Internal load balancer memory requirements are impacted by the number of threads, response time latency, and message sizes. The following internal load balancer memory allocations are recommended:
+
+* .5 GB (Default): Fewer than 500 simultaneous active connections.
+* 1.5 GB (Large): One or both of the following scenarios:
+
+** 500 or more simultaneous active connections.
+** Security policies enabled.
+
+[NOTE]
+These are general guidelines, and individual environments might require adjustments.
```

Comment:
Adjustments to what: guidelines or the amount of memory allocated?
In either case, it doesn't seem as though this sentence is needed.
"General" implies that the guidelines won't fit every case exactly.